flux-driver/src/collector.rs

```diff
@@ -93,6 +100,15 @@ impl<'tcx, 'a> SpecCollector<'tcx, 'a> {
                     let attrs = tcx.hir().attrs(hir_id);
                     let _ = collector.parse_tyalias_spec(item.def_id, attrs);
                 }
+                ItemKind::Const(_ty, _body_id) => {
+                    let hir_id = item.hir_id();
+                    let attrs = tcx.hir().attrs(hir_id);
+                    let span = item.span;
+                    let _ = match eval_const(tcx, item) {
```

Comment:
I would push this call to `eval_const` and all the error handling to `parse_const_spec`